### ISSUE TITLE
[Error: Hardware GPS triggers safety mode: No INS status message.] when turning on the SimControl Mode in DreamView

### ISSUE THREAD

**Issue by lygincn** [ApolloAuto/apollo#5724]
### System information

- **OS Platform and Distribution: Linux Ubuntu 14.04**
- **Apollo installed from**: https://github.com/ApolloAuto/apollo
- **Apollo version (1.0, 1.5, 2.0, 2.5, 3.0)**: 3.0

### Steps to reproduce the issue:

1. play demo_2.0.bag in DreamView
2.  swich on the SimControl button 
3. in the console errors reported: `Hardware GPS triggers safety mode: No INS status message.`

### Supporting materials (screenshots, command lines, code/script snippets):
![screenshot from 2018-09-10 16 59 50](https://user-images.githubusercontent.com/40354939/45287926-0f95fe00-b51c-11e8-9a43-0c1430d4087e.png)

After the error occurred the display broke down. 
Can you guys please tell me the possible reasons for this problem? Thank you a lot!

**Comment by Capri2014** [comment]
lagging of INS status in GPS message, are you running the 2.0 bag with LATEST MASTER HEAD?

**Comment by lygincn** [comment]
I checked out the messages in the demo_2.0.bag, there are already INS status massages published... like in the following pic:

![screenshot from 2018-09-11 09 52 57](https://user-images.githubusercontent.com/40354939/45333644-e4f28680-b5a9-11e8-8aae-92c133c35779.png)
**but** I have noticed that the frequency of the InsStat messages is lower than that of the other messages, not sure if this is relevant...
@Capri2014 I am not sure about the meaning of LATEST MASTER HEAD. Does it mean the latest source code or something? Thank you a lot!
